Split virology out of #349 (#396)
greenelab/covid19-review: content/02.introduction.md

Coronavirus genomes range from 27 to 32 kilobases (Kb) in length.
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
There are several fundamental genomic characteristics shared by all viruses in the *Nidovirales* order, including enveloped, non-segmented, and positive-sense (ssRNA+) genomes and a large number of non-structural genes originating through ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
Genome organization is highly conserved within the order [@doi:10.1007%2F978-1-4939-2438-7_1].
Coronavirus genomes are comprised of a replicase gene that is two-thirds of the genome.
This polypeptide is translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent), which form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
<!-- To do: Add a figure showing the structure of the genome.-->

> <!-- Also, everything focuses on S! Should we also talk about others?-->

Author: That definitely makes sense! It might be worth looping in @nilswellhausen who has offered to lead the charge on scientific visualizations -- if we include a diagram, what complementary information would be helpful to have in the text?

> <!-- Also, everything focuses on S! Should we also talk about others?-->

Reviewer: I can make a more visually appealing version of this published genome which is very thorough from a scientific standpoint: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7161481/

> <!-- Also, everything focuses on S! Should we also talk about others?-->

Reviewer 2: That sounds great. I think based on the information in the figure linked here, I think what we have in the text is sufficient..

Coronaviruses are classified into four genera: alpha, beta, delta and gamma coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species and delta coronaviruses infect both mammalian and avian species [@doi:10.1146/annurev-virology-110615-042301].
The viruses were initially subdivided into these genera based on antigenic relationships of the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins but are now divided by phylogenetic clustering.
[this section may be better as a figure panel + legend]
Phylogenetic analysis of a PCR amplicon fragment from five patients along with the total virus genome of 29.8 kilobases indicates that the virus is a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus.

> The sarbecovirus lineage also includes the human SARS coronavirus [@doi:10/ggjs7j].

Reviewer 2: I cannot seem to find this doi...

Author: It's a short DOI -- manubot doesn't like parentheses in citations but The Lancet loves them :) If you see something like that again, you can use doi.org/10/ggjs7j -- but [here is the link](https://www.thelancet.com/pdfs/journals/lancet/PIIS0140-6736(20)30154-9.pdf) to this paper! They are generated by shortdoi.org, in case you ever need to know how to do it!

Finally, the dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].

Thus, most of the viral entry process is determined by the viral genome, but the initial step, when the virus binds to a host receptor, also depends on the virus being able to recognize the host receptor.
Coronaviruses such as SARS-CoV, SARS-CoV-2, and MERS-CoV may therefore be engaged in an evolutionary arms race with the human immune system [@doi:110.1101/2020.05.13.093658; @doi:10.1101/2020.05.14.09613; @doi:10.1186/1743-422X-10-304].
Two "hot spots" within human ACE2 have been identified as driving the strong binding affinity between this receptor and two sites in SARS-CoV [@doi:10.1146/annurev-virology-110615-042301].
Coronaviruses bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].

> The S1 domain, which contains the RBM, evolves more rapidly than the S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].

Author: ```suggestion
The S1 domain of the spike protein, which contains the RBM, evolves more rapidly than _S_'s S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].
```

The possibility of asymptomatic cases is also being investigated.
The transmission and mortality rate estimations of COVID-19 remain to be determined.

#### Transmission and Evolution

SARS-CoV-2 is a novel human coronavirus, following SARS-CoV and MERS-CoV to become the third coronavirus causing severe pneumonia in humans to emerge in the 21st century [@doi:10.1016/bs.aivir.2018.01.001]. 

> The evolutionary origins of the SARS-CoV-2 virus are not yet fully understood.

Reviewer 2: Earlier it says that SARS-CoV2 is from the sarbecovirus lineage...I am assuming evolutionary lineage to mean phylogenetic lineage. Do you mean something else here?

https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1.abstract

Author: I didn't write this text but I'm inclined to agree with you! I think they meant the origins re: intermediate hosts. What about:
```suggestion
The proximal origins of the SARS-CoV-2 virus are not yet fully understood, but genomic analyses have placed it within the coronavirus phylogeny.
```

Reviewer 2: This might just be me, but I'm not sure what "proximal origin" means in this context, so the only edit I would make is to just have `The intermediate host of the SARS-CoV-2...` But I default to you as you have been reading WAY more on this subject

Reviewer: I think it refers to the most recent branch in which you can map an organism too. For example the most distal origin of humans would be the last common universal ancestor and the most proximal would be homo.

Author: ```suggestion
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
```
@ajlee21 I was going for what @nilswellhausen described, but it would be better if this is broadly understandable! I think this sentence is a bit clunky, but does it do a better job of getting the idea across (for now?)

Approximately one-third of common cold infections in humans is attributable to four out of six previously known human coronaviruses (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that are globally circulating in the human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81;@doi:10.1038/s41579-018-0118-9].

The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
The possibility of asymptomatic cases is also being investigated.

> The transmission and mortality rate estimations of COVID-19 remain to be determined.

Reviewer 2: Just curious, I'm not sure what data is publically available, but is this because of the inconsistency in testing? Not sure if we want to mention the reason here...

Author: I didn't write this section and agree we may be able to update!

Reviewer: I think we should definitely update that and provide a number. As it says it is "only" an estimation and hence only attempts to figure out the rates.

Author: I think we may be able to cut this line based on @alavendelm and @rdvelazquez section in Pathogenesis -- curious if they would agree?

Reviewer 3: The [reproduction number section](https://greenelab.github.io/covid19-review/#reproduction-number-and-dynamics-of-transmission) touches on transmission rate but doesn't address mortality rate. That section states, "Estimates of R0 for COVID-19 lie in the range R0=1.4-6.5".

I would be ok with just removing this line or stating the estimated range of R0 and referencing the Reproduction number section.

Reviewer 4: Yes agree. Good estimates are now available for transmission and mortality (tho we don't cover mortality in R0 section). Either remove this sentence, or we can add more details.

#### Cell Entry and Replication

Coronavirus virions are spherical with diameters ranging from 100 to 160 nanometers (nm).
The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating a distinctive "crown" shape for which the family of viruses was named [@isbn:0781760607; @doi:10.1128/JVI.80.3.1302–1310.2006].
The replication process is initiated by the viral spike protein: first, the virus binds to a host cell, and then the viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm.
The spike of SARS-CoV is structured in three segments: the ectodomain, a transmembrane anchor, and an intracellular tail [@doi:10.1128/JVI.02615-14].
The ectodomain contain two subdomains known as the S1 and S2 subunits, with the S1 (N-terminal) subunit guiding the binding of the virus to a host cell receptor and the S2 (C-terminal) subunit guiding the fusion process [@doi:10.1128/JVI.02615-14].

> <!-- Is it worth going into more detail about siganl peptides and secretory pathways? Not sure whether this is relevant information.-->

Reviewer 2: I think if there are treatments or parthenogenesis traits, mentioned in the other sections, that need that background it might be good to add

Author: That makes total sense @ajlee21! I might leave the comment in for now just as a reminder to check against the therapeutics section!

Finally, the dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].

Thus, most of the viral entry process is determined by the viral genome, but the initial step, when the virus binds to a host receptor, also depends on the virus being able to recognize the host receptor.
Coronaviruses such as SARS-CoV, SARS-CoV-2, and MERS-CoV may therefore be engaged in an evolutionary arms race with the human immune system [@doi:110.1101/2020.05.13.093658; @doi:10.1101/2020.05.14.09613; @doi:10.1186/1743-422X-10-304].
Two "hot spots" within human ACE2 have been identified as driving the strong binding affinity between this receptor and two sites in SARS-CoV [@doi:10.1146/annurev-virology-110615-042301].
Coronaviruses bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].

> The S1 domain, which contains the RBM, evolves more rapidly than the S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].

Reviewer 2: I think this line is redundant to the one below

Author: Ah, so it's actually two separate comparisons: S1 vs S2 and then S1-NTD vs S1-CTD. I may have gotten a little over-enthusiastic in this part since it was evolutionary!

> However, even within the S1 domain there is variability in conservation.

Reviewer 2: Not sure what this sentence means?

> However, even within the S1 domain there is variability in conservation.
> The receptors in the N-terminal domain (NTD) evolve more rapidly than those in the C-terminal domain (CTD) [@doi:10.1016/j.jmb.2020.04.009]. 

Author: How about this instead?
```suggestion
However, even within the S1 domain, some regions are more conserved than others, with the receptors in the N-terminal domain (S1-NTD) of S1 evolving more rapidly than those in its C-terminal domain (S1-CTD) [@doi:10.1016/j.jmb.2020.04.009].
```

> Both the NTD and CTD are involved in receptor binding and can function as receptor binding domains (RBDs) to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].

Reviewer 2: I think the naming may be a little confusing here. So earlier you say `S1 (N-terminal) subunit and...S2(C-terminal) subunit` so I interpreted that to mean that S1=N-terminal and S2-C-terminal. But now you are talking about S1 N-terminal and S1 C-terminal.

> Both the NTD and CTD are involved in receptor binding and can function as receptor binding domains (RBDs) to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].

Author: ```suggestion
Both S1-NTD and S1-CTD are involved in receptor binding and can function as receptor binding domains (RBDs) to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].
```

Approximately one-third of common cold infections in humans is attributable to four out of six previously known human coronaviruses (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that are globally circulating in the human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81;@doi:10.1038/s41579-018-0118-9].

The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
The possibility of asymptomatic cases is also being investigated.

> The transmission and mortality rate estimations of COVID-19 remain to be determined.

Reviewer 2: ```suggestion
```
